### PR TITLE
#26: Don't require -SNAPSHOT changes files

### LINKED ISSUE #26: Don't require -SNAPSHOT changes files
remove `-SNAPSHOT` from version number when checking the existence of changes files.

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #26

### CODE REVIEW

File: src/main/java/com/exasol/projectkeeper/validators/ChangelogValidator.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
  * Validator that checks the existence of the doc/changes/changes_X.X.X.md file for the current project's version.
  */
 public class ChangelogValidator implements Validator {
+    public static final String SNAPSHOT_ENDING = "-SNAPSHOT";
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/projectkeeper/validators/ChangelogValidator.java:21]: SNAPSHOT_SUFFIX maybe?

File: src/test/java/com/exasol/projectkeeper/validators/ChangelogValidatorTest.java [hunk 2]
```diff
@@ -20,17 +20,25 @@
 import org.junit.jupiter.api.io.TempDir;
 
 class ChangelogValidatorTest {
+    private static final String V_1_2_3 = "1.2.3";
```
Comment (AnastasiiaSergienko) [inline src/test/java/com/exasol/projectkeeper/validators/ChangelogValidatorTest.java:23]: Please, rename it. Just VERSION is enough, isn't it?

File: src/test/java/com/exasol/projectkeeper/validators/ChangelogValidatorTest.java [hunk 3]
```diff
@@ -20,17 +20,25 @@
...
 
     @Test
     void testValidation(@TempDir final File tempDir) {
-        final MavenProject project = createProject(tempDir);
+        final MavenProject project = createProject(tempDir, V_1_2_3);
+        assertThat(new ChangelogValidator(project),
+                hasValidationFindingWithMessage("E-PK-20 Could not find 'doc/changes/changes_1.2.3.md'."));
```
Comment (AnastasiiaSergienko) [inline src/test/java/com/exasol/projectkeeper/validators/ChangelogValidatorTest.java:29]: I think we agreed to use : after the code?

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

File: src/test/java/com/exasol/projectkeeper/validators/ChangelogValidatorTest.java (hunk 3 above)
Comment (jakobbraun) [reply src/test/java/com/exasol/projectkeeper/validators/ChangelogValidatorTest.java:29]: yes, we did but his one is older. (Even so the test is new...) So I will fix it with https://github.com/exasol/project-keeper-maven-plugin/issues/30

File: src/main/java/com/exasol/projectkeeper/validators/ChangelogValidator.java [hunk 4]
```diff
@@ -46,6 +47,15 @@ public ChangelogValidator validate(final Consumer<ValidationFinding> findingCons
         return this;
     }
 
+    private String getVersionWithoutSnapshotEnding() {
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/projectkeeper/validators/ChangelogValidator.java:50]: Also rename ending to suffix please

**Review by AnastasiiaSergienko — APPROVED** [review]
I think you forgot to rename the method, but everything else is fine

**Pull request merged**